apps/deploy_monitoring/docker-compose.tezedge_and_explorer.yml

```diff
@@ -2,8 +2,8 @@ version: "3"
 
 services:
   tezedge-node:
-    image: simplestakingcom/tezedge:latest-release
-    command: ["--network", "${TEZOS_NETWORK}", "--log", "terminal", "file", "--log-file", "/tmp/tezedge/tezedge.log", "--one-context", "--actions-store-backend", "none", "--peer-thresh-low", "30", "--peer-thresh-high", "45", "--p2p-port", "19732"]
+    image: tezedge/tezedge:latest-release
```

Comment:
@tizoc 
if this is for mainnet.simplestaking.com, we should use here better exact tag version and also for explorer

Reply:
@bkontur done. And I am talking with @adonagy, it may be better to keep this version for now until the new tool has been tested more.